package dogrun2;

enum kItemType{
    kItemFeed = 0;
    kItemTrain = 1;
    kItemPlay = 2;
};

message UIElementFont {
    required string desc = 1;
    required string font = 2;
    required int32 pos_x = 3;
    required int32 pos_y = 4;
    required int32 font_size = 5;
    repeated int32 font_color = 6;
};

message UIElementIcon {
    required string icon = 1;
    required int32 pos_x = 2;
    required int32 pos_y = 3;
    required int32 scale = 4;
    
};

message UIItemConf {
    required UIElementFont title = 1;
    required UIElementIcon icon = 2;
    required UIElementFont icon_num = 3;
    optional UIElementFont speed = 4;
    optional UIElementFont exp = 5;
    optional UIElementFont strong = 6;
    optional UIElementFont intimacy = 7;
};

message UIItemConfArray {
    repeated UIItemConf conf = 1;
};
